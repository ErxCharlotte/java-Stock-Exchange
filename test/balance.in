ADD trader1 100.55
ADD trader2 20.00
ADD trader3 200.00
ADD trader4 300.00
BALANCE trader1
BALANCE trade100
EXIT