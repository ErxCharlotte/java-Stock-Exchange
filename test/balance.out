$ Success.
$ Success.
$ Success.
$ Success.
$ $100.55
$ No such trader in the market.
$ Have a nice day.